apps/site/src/app/(home)/sections/FAQ/FAQ.module.scss

```diff
@@ -1,121 +1,112 @@
 @use "zothacks-theme" as theme;
 @use "bootstrap-utils" as utils;
 
-.faq {
-	position: relative;
-	display: flex;
-	flex-direction: column;
-	width: 80%;
+$item-padding-y: 32px;
+$item-padding-x: 48px;
```

Comment:
Thought: the text looks a bit squished on smaller screen widths. Maybe decrease the padding for smaller screens?